src/app/data-planes/views/DataPlaneDetailView.vue

```diff
@@ -138,15 +138,33 @@
                       ]"
                       :key="meta.protocol"
                     >
-                      <!-- rx and tx are purposefully reversed to rx=tx and tx=rx here due to the direction of the traffic (downstream) -->
-                      <ServiceTrafficCard
-                        :protocol="meta.protocol"
-                        :rx="item[meta.protocol]?.[`${meta.direction}_cx_tx_bytes_total`] as (number | undefined)"
-                        :tx="item[meta.protocol]?.[`${meta.direction}_cx_rx_bytes_total`] as (number | undefined)"
-                        :requests="meta.protocol === 'http' ? ['http1_total', 'http2_total', 'http3_total'].reduce((prev, key) => prev + (item.http?.[`${meta.direction}_rq_${key}`] as (number | undefined) ?? 0), 0) : undefined"
+                      <template
+                        v-for="port in [
+                          item.name.split('_')[1],
+                        ]"
+                        :key="port"
                       >
-                        {{ item.name }}
-                      </ServiceTrafficCard>
+                        <template
+                          v-for="inbound in [
+                            props.data.dataplane.networking.inbounds.find(item => `${item.port}` === `${port}`),
```

Comment:
This is the bit I will change in another PR. We are picking off the `_portnumber` from the stat, then searching for that across all inbounds, for every inbound. Whilst we mostly have 1 inbound I'd rather do this differently as mentioned in the description for the PR.